destroy

```diff
@@ -16,10 +16,13 @@ if [ "x$1" != "x" ]; then
   kubectl delete deploy,pod,rs,svc,ing,ds -l "$1" -n "$KUBE_NAMESPACE"
 else
   echo "Removing all deployment objects"
-  kubectl delete deploy,pod,rs,svc,ing,ds -l "app=$CI_ENVIRONMENT_SLUG" -n "$KUBE_NAMESPACE"
+  kubectl delete deploy,pod,rs,svc,ing,ds,configmap -l "app=$CI_ENVIRONMENT_SLUG" -n "$KUBE_NAMESPACE"
   echo "We are deleting on the environment=$CI_ENVIRONMENT_SLUG label for review apps with workers. You should be able to ignore errors below."
   kubectl delete deploy,pod,rs,svc,ing,ds -l "environment=$CI_ENVIRONMENT_SLUG" -n "$KUBE_NAMESPACE" || true
 fi
 
 echo "Removing auto-generated secrets..."
-kubectl delete secret $KUBE_NAMESPACE-secrets-$STAGE -n "$KUBE_NAMESPACE" || true
\ No newline at end of file
+kubectl delete secret $KUBE_NAMESPACE-secrets-$STAGE -n "$KUBE_NAMESPACE" || true
+
+echo "Removing certificates..."
+kubectl delete certificate tls-$CI_ENVIRONMENT_HOSTNAME -n "$KUBE_NAMESPACE" || true 
```

Comment:
Not everybody will have the certificate named ```tls-$CI_ENVIRONMENT_HOSTNAME``` so we can't put that in here.

In my environment, my certificates have labels inherited from the ingress controller it was provisioned from. So, as long as the ingress is labeled correctly, I think you should just be able to add certificates to the delete command at line 19.

Reply:
Actually I take it back, it shouldn't go in that delete line, because not everything will have cert-manager, and that command would cause errors in destruction for people that didn't.

I think you want
kubectl delete certificate -l "app=$CI_ENVIRONMENT_SLUG" -n "$KUBE_NAMESPACE" || true

Reply:
I reviewed the certificate resource and it does have the "app=$CI_ENVIRONMENT_SLUG" label. That being the case, could "certificate" just be added to the deployment resources delete command on line 19?
`kubectl delete deploy,pod,rs,svc,ing,ds,configmap[add "certificate" here] -l "app=$CI_ENVIRONMENT_SLUG" -n "$KUBE_NAMESPACE"`

Reply:
After another review of your feedback I see that you answered that question already. I've moved the certificate deletion to it's own command using the app label to discover it.